### PR TITLE
DO NOT MERGE! Propose new table of contents

### PR DESCRIPTION
**Pull request by DebbieAtSeam**
This PR is _only_ intended to provide an easy way to diff the existing and newly-proposed tables of contents (ToCs).
This proposal contains extra, explanatory content that makes this file an invalid GitBook ToC.
Please do not merge this PR!

### CODE REVIEW

**Comment by sybohy** [comment]
What's ToC?

**Comment by DebbieAtSeam** [comment]
@sybohy: ToC = table of contents

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -53,7 +54,59 @@
...
+    * [Climate Setting Schedule Table](seam-components/react-components/climate-setting-schedule-table.md)
+    * [Climate Setting Schedule Details](seam-components/react-components/climate-setting-schedule-details.md)
+    * [Create Climate Setting Schedule Form](seam-components/react-components/create-climate-setting-schedule-form.md)
+    * [Update Climate Setting Schedule Form](seam-components/react-components/update-climate-setting-schedule-form.md)
+    * [Supported Devices Table](seam-components/react-components/supported-devices-table.md)
+    * [Seam Provider](seam-components/react-components/seam-provider.md)
+    * [Connect Account Button](seam-components/react-components/connect-account-button.md)
+  * [Make a Supported Devices Page](seam-components/make-a-supported-devices-page.md)
+* [Sandbox Devices and Systems](device-guides/sandbox-and-sample-data/README.md) <-- Are any brands missing?
+  * [Virtual 2N Intercoms](device-guides/sandbox-and-sample-data/2n-intercoms-sample-data.md)
```
Comment (seveibar) [inline docs/SUMMARY.md:87]: I understand the desire to have a clear name here, but I think virtual is an overloaded term and we'll benefit more from consistency in terminology or a more descriptive term. An alternative could be "Emulated"

```suggestion
  * [Sandbox 2N Intercoms](device-guides/sandbox-and-sample-data/2n-intercoms-sample-data.md)
```

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -1,42 +1,43 @@
...
-* [🎉 Seam API Docs](README.md)
+* [🎉 Seam Documentation](README.md)
 * [🚲 Quick Start](quickstart.md)
-* [🚀 Going Live!](going-live.md)
+* [🚀 Go Live!](going-live.md)
 
 ## 📖 Core Concepts
 
 * [Overview](core-concepts/overview.md)
+  * [Pricing and Billing](reference/help-center/how-often-am-i-billed-with-seam.md)
```
Comment (dawnho) [inline docs/SUMMARY.md:10]: Not sure if we should have this so high up. Also, is this an API Guide concern?

Comment (DebbieAtSeam) [reply docs/SUMMARY.md:10]: @dawnho: Where would you suggest placing this topic? Is it something that customers ask about often, or is pricing information not appropriate for the docs? Is billing focused on Seam _API_ usage (vs. Seam Components, etc.)? Thanks!

Comment (sybohy) [reply docs/SUMMARY.md:10]: I think pricing info isn't appropriate in the docs per say. that's something we need to address on the marketing website.

Comment (dawnho) [inline docs/SUMMARY.md:10]:
```suggestion
```

File: docs/SUMMARY.md [hunk 3]
```diff
@@ -1,42 +1,43 @@
...
 * [Seam Console](core-concepts/seam-console.md)
 * [Workspaces](core-concepts/workspaces/README.md)
   * [API Keys](core-concepts/workspaces/api-keys.md)
   * [Personal Access Tokens](core-concepts/workspaces/personal-access-tokens.md)
```
Comment (dawnho) [inline docs/SUMMARY.md:14]: Wondering if we should have an `Authentication` section instead for covering the different ways of connect to the Seam API

File: docs/SUMMARY.md [hunk 4]
```diff
@@ -1,42 +1,43 @@
...
     * [Lifecycle of Access Codes](products/smart-locks/access-codes/lifecycle-of-access-codes.md)
     * [Access Code Requirements for Door Locks](products/smart-locks/access-codes/access-code-requirements-for-door-locks.md)
     * [Streamlining Access Code Management from the Manufacturer App](products/smart-locks/access-codes/accommodating-manufacturer-mobile-apps.md)
-    * [Troubleshooting Access Code Issues](products/smart-locks/access-codes/troubleshooting-access-code-issues.md)
-    * [Backup Access Codes](products/smart-locks/access-codes/backup-access-codes.md)
-    * [Offline Access Codes](products/smart-locks/access-codes/offline-access-codes.md)
-    * [Migrating Existing Access Codes](products/smart-locks/access-codes/migrating-existing-access-codes.md)
-* [🏢 Access Systems](products/access-systems/README.md)
-  * [User Management](products/access-systems/user-management.md)
+    * [Handling Recurring Access Codes](reference/help-center/handling-recurring-access-codes.md)
```
Comment (dawnho) [inline docs/SUMMARY.md:33]: We can remove this article. We'll be shipping weekly schedules soon, which will be a better approach than what is detailed here.
```suggestion
```

File: docs/SUMMARY.md [hunk 5]
```diff
@@ -1,42 +1,43 @@
...
+    * [Managing Access Codes for Hospitality](device-guides/implementing-robust-access-code-management-for-guests.md)
```
Comment (dawnho) [inline docs/SUMMARY.md:34]: A lot of the material is covered in our main access codes section. Suggest that we delete this article.
```suggestion
```

File: docs/SUMMARY.md [hunk 6]
```diff
@@ -53,7 +54,59 @@
...
+    * [Update Access Code Form](seam-components/react-components/update-access-code-form.md)
+    * [Climate Setting Schedule Table](seam-components/react-components/climate-setting-schedule-table.md)
+    * [Climate Setting Schedule Details](seam-components/react-components/climate-setting-schedule-details.md)
+    * [Create Climate Setting Schedule Form](seam-components/react-components/create-climate-setting-schedule-form.md)
+    * [Update Climate Setting Schedule Form](seam-components/react-components/update-climate-setting-schedule-form.md)
+    * [Supported Devices Table](seam-components/react-components/supported-devices-table.md)
+    * [Seam Provider](seam-components/react-components/seam-provider.md)
+    * [Connect Account Button](seam-components/react-components/connect-account-button.md)
+  * [Make a Supported Devices Page](seam-components/make-a-supported-devices-page.md)
+* [Sandbox Devices and Systems](device-guides/sandbox-and-sample-data/README.md) <-- Are any brands missing?
```
Comment (dawnho) [inline docs/SUMMARY.md:86]: Perhaps a task for the future, but is there a reason why we are breaking all of these out into individual pages?

Stripe has all of its test data all in one page:
https://stripe.com/docs/testing

File: docs/SUMMARY.md [hunk 7]
```diff
@@ -53,7 +54,59 @@
 * [👂 Noise Sensors](products/noise-sensors/README.md)
   * [Configuring Noise Threshold Settings](products/noise-sensors/configure-noise-threshold-settings.md)
 
-## 📋 API Clients
+## 📖 Developer Components and Tools
```
Comment (dawnho) [inline docs/SUMMARY.md:57]:
```suggestion
## 📖 Developer Tools
```
Suggest this instead?

File: docs/SUMMARY.md [hunk 8]
```diff
@@ -53,7 +54,59 @@
...
+
+* [Overview](NEW TOPIC)
+* [Connect Webviews](core-concepts/connect-webviews/README.md)
+  * [Embedding a Connect Webview in Your App](core-concepts/connect-webviews/embedding-a-connect-webview-in-your-app.md)
+  * [Attaching Custom Data to the Connect Webview](core-concepts/connect-webviews/attaching-custom-data-to-the-connect-webview.md)
+  * [Customizing Connect Webviews](core-concepts/connect-webviews/customizing-connect-webviews.md)
+* [Webhooks](core-concepts/webhooks.md)
+* [🧱 Seam Components](seam-components/overview/README.md)
```
[the remark this answers is not in the document]
Comment (dawnho) [reply docs/SUMMARY.md:65]: We can add references to the Seam Components in the "Capability Guides" section.

Screen Shot 2024-01-23 at 4 32 21 PM

See how Stripe references their "Web Elements" within "Payments" guide above.

File: docs/SUMMARY.md [hunk 9]
```diff
@@ -145,40 +198,22 @@
     * [Get a Credential](api-clients/access-control-systems/credentials/get-credential.md)
     * [Delete a Credential](api-clients/access-control-systems/credentials/delete-credential.md)
 
-## 🧱 Seam Components
+## Reference <-- Delete?
```
Comment (dawnho) [inline docs/SUMMARY.md:201]: Yeah, let's delete!

File: docs/SUMMARY.md [hunk 10]
```diff
@@ -145,40 +198,22 @@
...
-* [Glossary](reference/glossary.md)
-* [Help Center](reference/help-center/README.md)
-  * [What is an Action Attempt, when is it used?](reference/help-center/what-is-an-action-attempt-when-is-it-used.md)
-  * [How often am I billed with Seam?](reference/help-center/how-often-am-i-billed-with-seam.md)
-  * [Handling Recurring Access Codes](reference/help-center/handling-recurring-access-codes.md)
+* [Overview](reference/help-center/README.md)
+  * [What is an Action Attempt, when is it used?](reference/help-center/what-is-an-action-attempt-when-is-it-used.md) <-- Is this topic still useful, or is it redundant with core-concepts/action-attempts.md?
+  * [How often am I billed with Seam?](reference/help-center/how-often-am-i-billed-with-seam.md) <-- Rename to "Pricing and Billing," expand (including free vs. paid tiers, etc.), and move to be a child topic of core-concepts/overview.md.
+  * [Handling Recurring Access Codes](reference/help-center/handling-recurring-access-codes.md) <-- Move to be a child topic of products/smart-locks/access-codes/README.md.
+  * [Rate Limits and Guardrails](device-guides/rate-limits-and-guardrails.md)
```
Comment (dawnho) [inline docs/SUMMARY.md:212]: Might be good to move to Developer Tools section

File: docs/SUMMARY.md [hunk 11]
```diff
@@ -198,55 +233,35 @@
...
+* [Get started with Wyze Locks](device-guides/get-started-with-wyze-locks.md) <-- Delete. Redirect URL if needed for SEO.
+* [Reconnecting an Account](device-guides/reconnecting-an-account.md) <-- Move to be a child topic of new Connected Accounts topic.
+* [Access Code Management for Hospitality](device-guides/implementing-robust-access-code-management-for-guests.md) <-- Move to be a child topic of api-clients/access-codes/README.md.
+* [Rate Limits and Guardrails](device-guides/rate-limits-and-guardrails.md) <-- Move to be a child topic of reference/help-center/README.md?
 
 ## 🌉 Seam Bridge
 
-* [Connecting the Seam Bridge to DoorKing](seam-bridge/connecting-with-doorking.md)
+* [Overview](NEW TOPIC)
+* [Connecting the Seam Bridge to DoorKing](seam-bridge/connecting-with-doorking.md) <-- Is the Seam Bridge only applicable to DoorKing?
```
Comment (dawnho) [inline docs/SUMMARY.md:260]: We can remove this Seam Bridge section. This is old, and no longer used, and conflicts with the new Seam Bridge product.

File: docs/SUMMARY.md [hunk 12]
```diff
@@ -198,55 +233,35 @@
...
 
-## For Device Manufacturers
+## Device Manufacturer Advice or Device Manufacturer Guidance or Device Manufacturer Help
```
Comment (dawnho) [inline docs/SUMMARY.md:262]: I like "Device Manufacturer Guidance"

File: docs/SUMMARY.md (hunk 2 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:10]: Removed in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 3 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:13]: Added Authentication section in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 4 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:32]: Removed in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 5 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:32]: Removed in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 7 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:54]: Changed to Developer Tools in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:62]: Changed to UI Components in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md [hunk 13]
```diff
@@ -53,7 +54,59 @@
...
+## 📖 Developer Components and Tools
+
+* [Overview](NEW TOPIC)
+* [Connect Webviews](core-concepts/connect-webviews/README.md)
+  * [Embedding a Connect Webview in Your App](core-concepts/connect-webviews/embedding-a-connect-webview-in-your-app.md)
+  * [Attaching Custom Data to the Connect Webview](core-concepts/connect-webviews/attaching-custom-data-to-the-connect-webview.md)
+  * [Customizing Connect Webviews](core-concepts/connect-webviews/customizing-connect-webviews.md)
+* [Webhooks](core-concepts/webhooks.md)
+* [🧱 Seam Components](seam-components/overview/README.md)
+    * [Getting Started] (NEW TOPIC)
```
[the remark this answers is not in the document]
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:63]: Agreed. Omitted from [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 1 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:84]: Changed to Sandbox 2N Intercoms, etc. in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 9 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:198]: Deleted in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md [hunk 14]
```diff
@@ -145,40 +198,22 @@
...
-## Reference
+## Developer Advice or Developer Corner or Developer Help <-- For less-formal, "Help Center" types of topics. Topics only go here if they are truly not "baked" enough to go anywhere else or if there is truly no other relevant parent section.
 
-* [Device Provider Metadata](reference/device-provider-metadata.md)
-* [Glossary](reference/glossary.md)
-* [Help Center](reference/help-center/README.md)
-  * [What is an Action Attempt, when is it used?](reference/help-center/what-is-an-action-attempt-when-is-it-used.md)
-  * [How often am I billed with Seam?](reference/help-center/how-often-am-i-billed-with-seam.md)
-  * [Handling Recurring Access Codes](reference/help-center/handling-recurring-access-codes.md)
+* [Overview](reference/help-center/README.md)
```
[the remark this answers is not in the document]
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:205]: Removed in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 10 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:209]: Moved in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md [hunk 15]
```diff
@@ -198,55 +233,35 @@
...
+* [Get started with Minut Sensors](device-guides/get-started-with-minut-sensors.md) <-- Delete. Redirect URL if needed for SEO.
+* [Get started with 2N Intercoms](device-guides/get-started-with-2n-intercoms.md) <-- Delete. Redirect URL if needed for SEO.
+* [Get started with SmartThings Hubs + Smart Locks](device-guides/get-started-with-smartthings-hubs-+-smart-locks.md) <-- Add device guide. Then... Delete. Redirect URL if needed for SEO.
+* [Get started with Salto Locks](device-guides/get-started-with-salto-locks.md) <-- Delete. Redirect URL if needed for SEO.
+* [Get started with TTLock Devices](device-guides/get-started-with-ttlock-devices.md) <-- Delete. Redirect URL if needed for SEO.
+* [Get started with NoiseAware Sensors](device-guides/get-started-with-noiseaware-sensors.md) <-- Delete. Redirect URL if needed for SEO.
+* [Get started with Igloohome Locks](device-guides/get-started-with-igloo-locks.md) <-- Delete. Redirect URL if needed for SEO.
+* [Get started with Wyze Locks](device-guides/get-started-with-wyze-locks.md) <-- Delete. Redirect URL if needed for SEO.
```
[the remark this answers is not in the document]
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:249]: Moved, not deleted in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

File: docs/SUMMARY.md (hunk 11 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:257]: Already deleted in previous PR [https://github.com/seamapi/api-docs/pull/230](https://github.com/seamapi/api-docs/pull/230).

File: docs/SUMMARY.md (hunk 12 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:259]: Changed to Device Manufacturer Guidance in [https://github.com/seamapi/api-docs/pull/235/files](https://github.com/seamapi/api-docs/pull/235/files).

**Pull request closed without merging**